compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -410,7 +410,6 @@
     TYPE_CAST_PARAM(3032),
     KEY_SPECIFIER(3033),
     EXPLICIT_TYPE_PARAMS(3034),
-    ERROR_TYPE_PARAMS(3035),
```

Comment:
Shouldn't we change the tag for the subsequent kinds?

Reply:
+1